test/evergreen.yml

```diff
@@ -544,6 +559,7 @@ functions:
         remote_file: wiredtiger/${build_variant}/${revision}/artifacts/${task_name}_${build_id}${postfix|}.tgz
     # We remove the wiredtiger directory here to avoid to getting archived again by post tasks.
     - command: shell.exec
+      type: setup 
```

Comment:
On re-read this needs to be `test`. When we run cppsuite above the command always succeeds and saves the exit code to file. If cppsuite fails this is the command that will detect and report it.

Reply:
updated here [e83fa92](https://github.com/wiredtiger/wiredtiger/pull/9662/commits/e83fa924a0362168d2d671df5d49245ffea45907)